README.md

```diff
@@ -1,41 +1,49 @@
 # BastCastle Simple Home Automation using Home Assistant
 
 ## The haHA Gateway Drug
-As a part of a cloud computing practicum, a team of six came together to build a home automation process using Home Assitant for three specific use cases: locks, curtains, and vents. We started testing with lights, so we kept those scipts in for funzies. Further, we built our process in the cloud but optimized it for migration to a PI-Hole. 
+As a part of a cloud computing practicum, a team of six came together to build a home automation
```

Comment:
I'd suggest making this more like the STAR method and giving a little more background explanation of what our project prompt was. Maybe post Kyle's initial project in the wiki and link it?